corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -140,7 +140,21 @@
                 $('form[name="exportFiltersForm"]'));
             $scope.prepareExportError = null;
             $scope.preparingExport = true;
-            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {"Export type": $scope.exportList[0].export_type});
+            var userTypes = hqImport("hqwebapp/js/initial_page_data").get('user_types');
+            function getFilterName(exportType) {
+                return (exportType === "form" ? "emw" : "case_list_filter");
+            }
+            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {
+                "Export type": $scope.exportList[0].export_type,
+                "filters": _.map(
+                    $("#exportFiltersFormId").find("input[name="
+                        + getFilterName($scope.exportList[0].export_type) + "]")
+                        .val().split(','),
+                    function (item) {
+                        if (item.substring(0,3) === "t__") { return userTypes[item.substring(3)]; }
```

Comment:
Are you attached to having the if block on the same line as the if? I think single-line ifs are harder to read. But I don't think we have an actual best practice, so we can also just chalk that up to stylistic differences.

Reply:
I'm tempted to make some picky comments about the fragility of hard coding indices (I like to do things like make `t__` a variable and then use `item.substring(prefix.length)` instead of `item.substring(3)`, because they're less liable to break because someone changed the magic string but not the magic numbers).

Alternatively, this function could be replaced with `return item.replace(/^t__/, '')`